src/services/identity/LaunchesService.ts

```diff
@@ -65,22 +64,38 @@ export class LaunchesService {
     this.redirectionsRepository = redirectionsRepository
   }
 
-  create = async (createParams: SiteLaunchCreateParams): Promise<Launch> => {
-    const createLaunch = await this.launchesRepository.create(createParams)
+  createOrUpdate = async (
+    createParams: SiteLaunchCreateParams
+  ): Promise<Launch> => {
+    const [launch] = await this.launchesRepository.upsert(createParams)
+
+    // In the case that this is a re-launch,
+    // need to delete previous records in redirection table
+    const outDatedRedirections = await this.redirectionsRepository.findAll({
```

Comment:
if my understanding is correct and we want to check if this is a re-launch, we can use the [second return value](https://sequelize.org/api/v6/class/src/model.js~model#static-method-upsert) of upsert (returns `true` if newly created, `false` otherwise)

Reply:
Hmm this was by design!
my thinking was that if in the future we had support for all sorts of redirections (excluding the standard www) ones, we could have a trail for when the record existed in our system and when it was removed, and when a new record was created. 
I dont have strong opinions on this, neither are we sure this information will be useful to out users in the cms in the future, and if you feel this info is too verbose, we can stick to just upsert rather than creating another entry altogether.

Reply:
my concern was that lines 74 onwards run even when it's newly created HAHA but i guess it's ok to ignore this for now

Reply:
fair - c783ad6